app/config/config.yml

```diff
@@ -157,11 +157,11 @@ nelmio_cors:
             max_age: 3600
         '^/':
             #origin_regex: true
-            allow_origin: ['^http://localhost:[0-9]+']
+            allow_origin: ['*']
             allow_headers: ['Authorization','content-type']
             allow_methods: ['POST', 'PUT', 'GET', 'DELETE']
             max_age: 3600
-            hosts: ['^api\.']
+            #hosts: ['^api\.']
```

Comment:
Why should we allow cors on the UI? 🤔

Reply:
As we still don't have API endpoints for clients information, I want to get application info right from the ui, like native applications do. If you thind it is unsecuse, let us kill last commit.

Reply:
By client info, you mean the credentials?
I don't really like that way to do. It'll work for you, but it'll also open other security issues.

Reply:
not password/username but clientId and clientSecret

Reply:
And then - CORS restriction works only for web apps, any local programs, even **curl** or **wget** can grab pages easily.